src/js/store/factory/Guide.js

```diff
@@ -46,7 +98,7 @@ var def = module.exports = function(gtype, type, scale) {
     };
   }
 
-  guide.title = undefined;
+  guide.title = 'title';
```

Comment:
Revert to `undefined`? Titles get automatically filled in during data binding and, if a user were to create an axis manually, I'd prefer for the title to be blank rather than `title`.